test/util/execute/discovery.go

```diff
@@ -0,0 +1,12 @@
+package execute
```

Comment:
Can this be moved in the `discovery` package so it's consistent with the other test suites?

Reply:
removed the file, it should only come in a following PR

Reply:
Just curious, why can't you add it as part of this?

Reply:
This code would be unused by this patch.
It's better to add it in the PR that will actually use it.